packages/cli/test/setup.js

```diff
@@ -0,0 +1,4 @@
+const chalk = require('chalk');
+
+// Disable logging coloring on all environments
+chalk.level = 0;
```

Comment:
Unit tests were not working on local because they were printing colors, and the assertions didn't work.